### DISCUSSION TITLE
Issues opening provided link to National Grasslands Units!

### DISCUSSION THREAD

**Discussion by JulesMRuef** [earthlab-education/Earth-Analytics-2023-01-Intro#681] [! GitHub help]
Hi there! Is anyones else having issues getting theNational Grasslands Unit link to work? 

(https://data.fs.usda.gov/geodata/edw/edw_resources/shp/S_USA.NationalGrassland.zip) 

It seemed to be working fine yesterday but when I restarted my kernel this morning and ran my code, I got all kinda of errors! I tried opening this up in my web browser and through the US Forest Service website to no avail. 

<img width="1218" alt="Screen Shot 2023-12-16 at 12 15 10 PM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/143224732/cf785b7b-9801-49f6-ad43-7f13646424a6">

**Comment by eculler** [comment]
You can access the data temporarily here: https://github.com/eculler/data-for-bootcamp/releases/download/humboldt-park/us_national_grasslands.zip

@earthlab-education/ea-students
